backend/Exceptions/Validation/InvalidRangeValidationException.php

```diff
@@ -6,12 +6,13 @@
 
 use Throwable;
 
-class InvalidCountValidationException extends ValidationException
+class InvalidRangeValidationException extends ValidationException
 {
     public function __construct(
         int $entryID,
```

Comment:
We use camelCase, don't we? S it should be `entryId`.

Reply:
Yes! I still didn't manage to track each of these, and I didn't figure out yet how to find stuff using regex/something similar.

Reply:
In PHPStorm: https://www.jetbrains.com/help/phpstorm/searching-everywhere.html